src/server/controllers/search.js

```diff
@@ -0,0 +1,13 @@
+
+
+const searchRecipesByName = async (req, res, next) => {
+  try {
+    const name = req.query.name;
+    const recipes = await Recipe.find();
```

Comment:
Declare helpful function (in folder `server/routes/recipes`) in file `recipesRoutes.js`. It will be more controled if we have all controll routes in same file.